src/components/StudyView/StudyView.tsx

```diff
@@ -159,6 +250,16 @@ const StudyView: React.FC<StudyViewProps> = ({ submissionId }) => {
                 : undefined
             }
           />
+
+          {/* TODO: lock/unlock submission when opening/closing the modal */}
```

Comment:
I will assume this is a `TODO` for the future (after this branch gets merged in) as opposed to before this branch gets merged in.

Reply:
Correct